### PR TITLE
update serializers without viewset

### PR DESCRIPTION
**Pull request by Artanic30**

### CODE REVIEW

File: backend/lin/serializers.py [hunk 1]
```diff
@@ -15,7 +18,9 @@ def create(self, validated_data):
 
     def update(self, instance, validated_data):
```
Comment (harveyqiu) [inline backend/lin/serializers.py:20]: 这种override的，就不用这么写了
写个print 
下面super不就好了

Comment (Artanic30) [reply backend/lin/serializers.py:20]: 174行的那个问题呢

File: backend/lin/views.py [hunk 2]
```diff
@@ -137,39 +136,44 @@ def profile(request, name):
...
     board = checking_board(request.user, board_id)
+    se_board = BoardSerializers(board)
     data = {}
     if board:
-        sign = board.sign
-        if sign == BoardInfo.objects.get(board=board, players=request.user).sign:
-            b = ChessBoard(exist_board=json.loads(board.content))
+        sign = se_board.data.get('sign')
+        tem_boinfo = BoardInfo.objects.get(board=board, players=request.user)
+        # todo: is serializer necessary if we could get the data through models directly?
```
Comment (harveyqiu) [inline backend/lin/views.py:173]: not necessary

Comment (harveyqiu) [reply backend/lin/views.py:173]: use serializer when you need to response to users or validate data

File: backend/lin/serializers.py (hunk 1 above)
Comment (harveyqiu) [reply backend/lin/serializers.py:20]: reviewed

**Pull request merged**